src/airflow/factory/task_factory.py

```
@@ -0,0 +1,136 @@
+        _pool (str): O nome da pool de recursos do Airflow à qual a tarefa pertence.
+        _execution_timeout (timedelta | None): A duração máxima permitida para a execução da tarefa.
+        _retry_delay (timedelta): O atraso entre tentativas de reexecução da tarefa em caso de falha.
+        _kwargs (dict): Argumentos adicionais de palavra-chave passados diretamente para o ``PythonOperator``.
+    """
+    def __init__(
+        self,
+        task_id: str,
+        func: _Callable,
+        func_kwargs: dict[str, _Any] | None = None,
+        pool: str | None = None,
+        execution_timeout= {'hours' : 10},
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)